data-products/src/main/scala/org/sunbird/analytics/exhaust/OnDemandExhaustJob.scala

```diff
@@ -180,13 +182,20 @@ trait OnDemandExhaustJob {
       // $COVERAGE-OFF$ Disabling scoverage
       case "s3" =>
         CommonUtil.getS3File(storageConfig.container, "");
+      case "oci" =>
+        CommonUtil.getS3File(storageConfig.container, "");
```

Comment:
@ddevadat Since statements are same, multiple conditions can be passed as below.
```
case "s3" | "oci" =>
```